analyzers/tests/SonarAnalyzer.UnitTest/TestCases/UnusedPrivateMember.CSharp10.cs

```diff
@@ -12,9 +12,9 @@ public record struct RecordStruct
 
         private nint Value { get; init; } = 42;
 
-        private nint UnusedValue { get; init; } = 42; // FN
+        private nint UnusedValue { get; init; } // Noncompliant
```

Comment:
I've noticed the initializer value was removed. Does it matter in this case?

Reply:
The example with the initializer is a FN, I documented it `UnusedPrivateMemberTest.Properties.cs` and left the CSharp10 part only CSharp 10 related.